api-server/src/server/boot/authentication.js

```diff
@@ -87,6 +87,8 @@ module.exports = function enableAuthentication(app) {
     createGetPasswordlessAuth(app)
   );
 
+  api.get('/mobile-login', ifUserRedirect, mobileLogin(app));
```

Comment:
```suggestion
  api.get('/mobile-login', ifNotMobileRedirect, ifUserRedirect, mobileLogin(app));
```

You should create a handler called `ifNotMobileRedirect` that checks for the request from mobile devices (non-desktop browsers) and then reply with non-availability.

It could be user-agent-based for now.